src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/config": "^4.2|^5.0",
         "symfony/dependency-injection": "^4.3|^5.0",
         "symfony/http-foundation": "^4.3|^5.0",
-        "symfony/http-kernel": "^4.3|^5.0",
+        "symfony/http-kernel": "^4.3",
```

Comment:
why? `Tests/Kernel/ConcreteMicroKernel` should be updated instead and that should be enough, isn't it?

Reply:
should be reverted to me, as https://github.com/symfony/symfony/pull/31907 should fix the issue you're having